vermouth/ffoutput.py

```diff
@@ -0,0 +1,140 @@
+# Copyright 2020 University of Groningen
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Write .ff files.
+
+The FF file format describes molecule components for a given force field.
+
+The format is built on top of a subset of the ITP format. Describing a block
+is done in the same way an ITP file describes a molecule.
+"""
+
+class ForceFieldDirectiveWriter():
+    """
+    Write force-field files according to the
+    vermouth force-field definition.
+    """
+    def __init__(self, forcefield, stream):
+        """
+        Parameters
+        ----------
+        forcefield: `:class:vermouth.forcefield.ForceField`
+            the force-field object to write
+
+        stream: ``
+            the stream to which to write; must have a write method
+        """
+        self.forcefield = forcefield
+        self.stream = stream
+        # these attributes have a specific order in the moleculetype section
+        self.normal_order_block_atoms = ["atype", "resid", "resname",
+                                         "atomname", "charge_group", "charge", "mass"]
```

Comment:
Maybe these should be class attributes/variables rather than instance attributes/variables.